nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -140,6 +140,50 @@ let
         ssl_stapling_verify on;
       ''}
 
+      ${optionalString (cfg.recommendedBrotliSettings) ''
+        brotli on;
+        brotli_static on;
+        brotli_comp_level 5;
+        brotli_window 512k;
+        brotli_min_length 256;
+        brotli_types
```

Comment:
Can you add a comment where we took those settings?

Reply:
Added comment.